types/meteor/accounts-base.d.ts

```diff
@@ -236,4 +236,20 @@ declare module "meteor/accounts-base" {
          */
         function _checkPassword(user: Meteor.User, password: Password): { userId: string; error?: any };
     }
+
+    module Accounts {
+        type StampedLoginToken = {
+            token: string;
+            when: Date;
+        }
+        type HashedStampedLoginToken = {
+            hashedToken: string;
+            when: Date;
+        }
+
+        function _generateStampedLoginToken(): StampedLoginToken;
+        function _hashStampedToken(token: StampedLoginToken): HashedStampedLoginToken;
+        function _insertHashedLoginToken(userId: string, token: HashedStampedLoginToken): void;
```

Comment:
According to the [source](https://github.com/meteor/meteor/blob/6e61b5bc9ca8a8170b5ef28832010d0c12c2f5f2/packages/accounts-base/accounts_server.js#L830-L841), there's also third argument named `query` and I think it'd be good to add it.

Reply:
Thanks, [fixed](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/50549/commits/f9ec1f94912836e722fb4d06b6db93aef997d202)